test/task.js

```diff
@@ -24,21 +24,28 @@ describe('task', function() {
   });
 
   it('should register a named function', function(done) {
-    taker.task(noop);
+    expect(taker.task('noop')).to.be.undefined();
+    expect(taker.task(noop)).to.be.undefined();
     expect(taker.task('noop')).to.equal(noop);
     done();
   });
 
   it('should register an anonymous function by string name', function(done) {
-    taker.task('test1', anon);
+    expect(taker.task('test1')).to.be.undefined();
```

Comment:
Because I wanted to check that `task` returned nothing when working as a setter.
Is it better to change back to original?